src/Components/App.js

```diff
@@ -2,44 +2,96 @@ import React from 'react';
 import './App.css';
 import Grid from './Grid';
 import GridHandler from '../Utils/GridHandler';
+import GameOfLifeSimulator from '../Utils/GameOfLifeSimulator';
 const autoBind = require('auto-bind');
 
+const StartSimulationText = 'Simulate';
+const StopSimulationText = 'Stop';
+
 class App extends React.Component {
   constructor(props){
     super(props);
     this.state = {
       width: 20,
-      height: 20
+      height: 20,
+      simulationRunning: false,
+      tickIntervalMillis: 500
     };
     this.widths = [10, 20, 30];
     this.heights = [10, 20, 30];
+    this.simulationIntervalId = 0;
+    this.simulator = new GameOfLifeSimulator();
     this.gridHandler = new GridHandler(this.state.width, this.state.height, () => this.setState(this.state));
     autoBind(this);
   }
   heightChange(event){
-    const value = parseInt(event.target.value)
+    const value = parseInt(event.target.value);
     this.setState({height: value});
     this.gridHandler.resetGrid(this.state.width, value, false);
   }
   widthChange(event){
-    const value = parseInt(event.target.value)
+    const value = parseInt(event.target.value);
     this.setState({width: value});
     this.gridHandler.resetGrid(value, this.state.height, false);
   }
+  tickIntervalChange(event){
+    const value = event.target.value;
+    this.setState({tickIntervalMillis: value}, () => {
+      if (this.state.simulationRunning){
+        this.stopSimulation();
+        this.startSimulation(value);
+      }
+    });
+  }
+  changeSimulationState(){
+    if (this.state.simulationRunning){
+      this.stopSimulation();
+    } else {
+      this.startSimulation(this.state.tickIntervalMillis);
+    }
+  }
+  startSimulation(intervalMillis){
+    this.gridHandler.setEnableTileUpdates(false);
+    this.setState({simulationRunning: true});
+    try {
+      this.simulationIntervalId = setInterval(() => {
```

Comment:
Have you tested this failing? You might want your try-catch inside the `setInterval` function

Reply:
I have not but I will add a ticket to add tests for that to check at a later date. Need to investigate how to setup tests for components first.